packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDeploySourcePath.test.ts

```diff
@@ -128,20 +128,6 @@ describe('Force Source Deploy Using Sourcepath Option', () => {
       expect(pollStatusStub.calledOnce).to.equal(true);
     });
 
-    it('componentSet should have sourceApiVersion set', async () => {
-      const executor = new LibraryDeploySourcePathExecutor();
-      const data = path.join(
-        workspaceUtils.getRootWorkspacePath(),
-        'force-app/main/default/classes/'
-      );
-      const continueResponse = {
-        type: 'CONTINUE',
-        data: [data]
-      } as ContinueResponse<string[]>;
-      const componentSet = executor.getComponents(continueResponse);
-      expect((await componentSet).sourceApiVersion).to.equal('11.0');
```

Comment:
No, because
1) this in an unigration test
2) we want to get rid of unigration test, not keep maintaining them
3) unit test coverage will be coming in the next PR (this PR is already large, and the next PR for unit test will have some refactoring work in it as well, so it's being split up)